src/main/java/com/powsybl/openloadflow/network/impl/Networks.java

```diff
@@ -58,11 +58,7 @@ private static double getDoubleProperty(Identifiable identifiable, String name)
 
     private static void setDoubleProperty(Identifiable identifiable, String name, double value) {
         Objects.requireNonNull(identifiable);
-        if (Double.isNaN(value)) {
-            identifiable.setProperty(name, null);
-        } else {
-            identifiable.setProperty(name, Double.toString(value));
-        }
+        identifiable.setProperty(name, Double.toString(value));
```

Comment:
I think we should keep the previous behaviour, so remove the property in case of NaN. I does not make sense to keep a NaN in the property and is not sync with getDoubleProperty

Reply:
In fact, I falled in a null pointer exception with the previous code because property cannot be null. So maybe, in case of Nan, we don't set the property ?

Reply:
Oups, identifiable.setProperty(name, null) should have been identifiable.remove(name)

Reply:
I have not this function remove from my identifiable :-)

Reply:
identifiable.getPropertyNames().remove(name) ?